src/navigation/MainNavigator.tsx

```
@@ -8,20 +8,79 @@ import UsersScreen from '../screens/UsersScreen';
 import SettingsScreen from '../screens/SettingsScreen';
 import {useAuth} from '../hooks/useAuth';
 import {useTranslationSafe} from '../hooks/useTranslationSafe';
+import {UserRole} from '../types';
```

ESLint

[error] 'UserRole' is defined but never used.

(@typescript-eslint/no-unused-vars)